decidim-comments/app/cells/decidim/comments/comment_cell.rb

```diff
@@ -45,6 +45,8 @@ def cache_hash
         hash.push(model.must_render_translation?(current_organization) ? 1 : 0)
         hash.push(model.authored_by?(current_user) ? 1 : 0)
         hash.push(model.reported_by?(current_user) ? 1 : 0)
+        hash.push(model.up_votes.count)
+        hash.push(model.down_votes.count)
```

Comment:
@andreslucena , those 2 keys are adding 2 additional count queries for each one of the comments being displayed. Apart of the code suggestion, would add some counter caching to it. 

```suggestion
        hash.push(model.up_votes.size)
        hash.push(model.down_votes.size)
```